⚠️ Nightly upstream-dev CI failed ⚠️

[Workflow Run URL](https://github.com/pydata/xarray/actions/runs/2402645617)
<details><summary>Python 3.10 Test Summary Info</summary>

```

```

</details>

The CI failures appear to be related to https://github.com/dask/dask/issues/9137

```bash
file /home/runner/work/xarray/xarray/xarray/tests/test_distributed.py, line 82
  @pytest.mark.parametrize("engine,nc_format", ENGINES_AND_FORMATS)
  def test_dask_distributed_netcdf_roundtrip(
file /usr/share/miniconda/envs/xarray-tests/lib/python3.10/site-packages/distributed/utils_test.py, line 138
  @pytest.fixture
  def loop(cleanup):
E       fixture 'cleanup' not found
>       available fixtures: add_standard_imports, backend, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, loop, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_netcdf_filename, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.
```

Ccing @jrbourbeau for visibility upstream...

--- a/xarray/tests/test_distributed.py
+++ b/xarray/tests/test_distributed.py
@@ -12,7 +12,7 @@ distributed = pytest.importorskip("distributed")  # isort:skip
 
 from dask.distributed import Client, Lock
 from distributed.client import futures_of
-from distributed.utils_test import cluster, gen_cluster, loop
+from distributed.utils_test import cluster, gen_cluster, loop, cleanup  # noqa: F401
 
 import xarray as xr
 from xarray.backends.locks import HDF5_LOCK, CombinedLock

